src/test/renderer.test.cpp

```diff
@@ -63,7 +56,7 @@ int main(int argc, char *argv[]) {
 
     // Create texture object.
     Vector3 texturePosition = Vector3(0, 0, 0);
-    TextureObject textureObject(texturePosition, "assets/test.jpg");
+    TextureObject textureObject(texturePosition, "assets/admirals.png");
```

Comment:
Need to change TextureObject::render() method to use the dimensions of the new image (was 1400x1400 px). While it will work without the change, it might be confusing in the future.